custom_components/tuya_local/devices/feyree_ev_charger.yaml

```diff
@@ -119,6 +119,8 @@ secondary_entities:
             value: "Socket leakage"
           - dps_val: 16
             value: "Socket temperature is to high."
+          - dps_val: 32
```

Comment:
Leave out the dps_val here to have a catch-all for undefined values. But if you are getting values that are outside the documented values, then your dps_val are likely wrong.  I see you have defined then as sequential integers. But if this is really a bitfield (bitmap in tuya docs), then they should all be powers of 2, so 32 should probably be "Control pilot line error"

Reply:
I see, you're probably right.